style.css

```diff
@@ -174,11 +174,11 @@ input[type="submit"],
 textarea:focus,
 .button,
 .gallery img {
-	-webkit-transition: all 0.1s ease-in-out;
-	-moz-transition:    all 0.1s ease-in-out;
-	-ms-transition:     all 0.1s ease-in-out;
-	-o-transition:      all 0.1s ease-in-out;
-	transition:         all 0.1s ease-in-out;
+	-webkit-transition: all .1s ease-in-out;
```

Comment:
Ok, it is in the WP standards, but personally, I chose to omit the zeros to make my CSS as clean as possible.